Octokit/Models/Request/InvitationUpdate.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Diagnostics;
+using System.Globalization;
+
+
+namespace Octokit
+{
+    /// <summary>
+    /// Used to update a invitation.
+    /// </summary>
+    /// <remarks>    
+    /// </remarks>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class InvitationUpdate
```

Comment:
I made a previous comment that the `permissions` are actually provided on the initial `Invite()` as well as on this update (and infact will also be added to the `Add()` method on the collaborators client, when that issue is fixed), so this class could probably be re-used for all cases.  This means it should be renamed to something like `CollaboratorRequest` rather than `InvitationUpdate`